### PR TITLE
Ensure config blocks are consistent

### PR DESCRIPTION
**Pull request by HeahDude**
Given some more love to 4.4 before it's EOL.

To avoid tons of conflict when merging upstream, if accepted, it should be reverted in 5.4, I'll send PRs dedicated to 5.4..

### CODE REVIEW

File: bundles/configuration.rst [hunk 1]
```diff
@@ -69,24 +76,28 @@ can add some configuration that looks like this:
...
+        namespace Symfony\Component\DependencyInjection\Loader\Configurator;
+
+        return static function (ContainerConfigurator $container) {
+            $container->extension('acme_social', [
+                'twitter' => [
+                    'client_id'     => 123,
+                    'client_secret' => 'your_secret',
+                ],
+            ]);
+        };
```
Comment (OskarStark) [inline bundles/configuration.rst:101]: @Nyholm any idea how to deal with this error?

Comment (Nyholm) [reply bundles/configuration.rst:101]: That error is the only one Im confident about. 

It just says that the "standard out-of-the-box symfony installation does not have a bundle that provides config for `acme`.  Which all make sense.

Ie, this error/warning is just to ignore.

**Comment by HeahDude** [comment]
I've missed those reported errors, at least some about XML are legit, I'll do a pass later to fix as many as I can.

**Comment by HeahDude** [comment]
As said Tobias, reported errors that remain are legit.

PR is ready.

**Comment by javiereguiluz** [comment]
Thanks a lot Jules!

**Pull request merged**